plugins/mcreator-localization/help/fr_FR/block/speed_factor.md

```diff
@@ -0,0 +1,4 @@
+Ce paramètre contrôle la vitesse des entités sur ce bloc.
+
+La valeur par défault utilisé par la majorité des blocs est 1.

```

Comment:
Same here. Kept the "défaut" in english, forgot the "é" for "utilisée" and add "de" before 1, and keep `1.0` as double.
Replace by this : `La valeur par défaut utilisée par la majorité des blocs est de 1.0.`